spec/services/dashboard_event_handler_spec.rb

```diff
@@ -95,6 +95,19 @@
       end
     end
 
+    context "saved with_state abandoned" do
```

Comment:
nit, can see this is used else where too but is it just a typo there and here `with_state` => `with state`